_posts/2018-12-27-dropping-puppet4.md

```diff
@@ -0,0 +1,22 @@
+---
+layout: post
+title: Dropping Puppet 4 support
+date: 2018-12-27
+github_username: bastelfreak
+---
+
+Puppet 4 will be end of life by the December 31 2018. There isn't an official
```

Comment:
"by December 31st 2018". 
Although I would be tempted to just use the ISO8601 date of 2018-12-31.
"by 2018-12-31"

Reply:
thanks, fixed it